+p_xml: 'version="1.0" encoding="UTF-8" standalone="yes" '
+directive: DOCTYPE bugzilla SYSTEM "http://bugzilla.fieldtriptoolbox.org/page.cgi?id=bugzilla.dtd"
bugzilla:
  +@version: 4.4.1
  +@urlbase: http://bugzilla.fieldtriptoolbox.org/
  +@maintainer: r.oostenveld@donders.ru.nl
  bug:
    bug_id: "183"
    creation_ts: 2010-10-18 10:31:00 +0200
    short_desc: include job number in rethrown error inside peerget
    delta_ts: 2011-01-05 12:01:07 +0100
    reporter_accessible: "1"
    cclist_accessible: "1"
    classification_id: "1"
    classification: Unclassified
    product: FieldTrip
    component: peer
    version: unspecified
    rep_platform: PC
    op_sys: Mac OS
    bug_status: CLOSED
    resolution: WONTFIX
    bug_file_loc:
    status_whiteboard:
    keywords:
    priority: P1
    bug_severity: enhancement
    target_milestone: '---'
    everconfirmed: "1"
    reporter:
      +content: r.oostenveld
      +@name: Robert Oostenveld
    assigned_to:
      +content: r.oostenveld
      +@name: Robert Oostenveld
    cc: a.stolk8
    comment_sort_order: oldest_to_newest
    long_desc:
      - +@isprivate: "0"
        commentid: "372"
        comment_count: "0"
        who:
          +content: r.oostenveld
          +@name: Robert Oostenveld
        bug_when: 2010-10-18 10:31:30 +0200
        thetext: "On 15 Oct 2010, at 22:27, Arjen Stolk wrote:\n\nHey Robert,\n \nIs het mogelijk om de 'jobnummer' mee te geven aan de error? Dan weet je in een oogopslag welke mislukt is.\n \nCiao,\nArjen"
      - +@isprivate: "0"
        commentid: "592"
        comment_count: "1"
        who:
          +content: r.oostenveld
          +@name: Robert Oostenveld
        bug_when: 2010-12-08 11:56:18 +0100
        thetext: if the peercellfun detects a failed job, it should print the job number so that the end-user can perform diagnosis
      - +@isprivate: "0"
        commentid: "681"
        comment_count: "2"
        who:
          +content: r.oostenveld
          +@name: Robert Oostenveld
        bug_when: 2010-12-15 20:50:01 +0100
        thetext: peerget does not know the job number, so cannot include it. Peercellfun would know the number, but that one is not rethrowing the error
      - +@isprivate: "0"
        commentid: "710"
        comment_count: "3"
        who:
          +content: r.oostenveld
          +@name: Robert Oostenveld
        bug_when: 2010-12-19 09:32:06 +0100
        thetext: peercellfun now catches the error (try-catch) and rethrows including the jobnumber
      - +@isprivate: "0"
        commentid: "721"
        comment_count: "4"
        who:
          +content: a.stolk8
          +@name: Arjen Stolk
        bug_when: 2010-12-20 08:17:46 +0100
        thetext: |-
          Nice!

          (In reply to comment #3)
          > peercellfun now catches the error (try-catch) and rethrows including the
          > jobnumber
      - +@isprivate: "0"
        commentid: "818"
        comment_count: "5"
        who:
          +content: r.oostenveld
          +@name: Robert Oostenveld
        bug_when: 2011-01-05 11:57:02 +0100
        thetext: selected a long list of resolved bugs from roboos and changed the status into "RESOLVED"
      - +@isprivate: "0"
        commentid: "897"
        comment_count: "6"
        who:
          +content: r.oostenveld
          +@name: Robert Oostenveld
        bug_when: 2011-01-05 12:01:07 +0100
        thetext: selected all old bugs from roboos with status RESOLVED and changed it into CLOSED
